### PR TITLE
Implement section 9 rules

### PR DESCRIPTION
**Pull request by jeniawhite**
### Summary of your changes
Added implementation for some of the section 9 rules

### Related Issues
- Related: https://github.com/elastic/cloudbeat/issues/1420

### Checklist
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary README/documentation (if appropriate)

#### Introducing a new rule?

- [x] Generate rule metadata using [this script](https://github.com/elastic/cloudbeat/tree/main/security-policies/dev#generate-rules-metadata)
- [x] Add relevant unit tests
- [ ] Generate relevant rule templates using [this script](https://github.com/elastic/cloudbeat/tree/main/security-policies/dev#generate-rule-templates), and open a PR in [elastic/packages/cloud_security_posture](https://github.com/elastic/integrations/tree/main/packages/cloud_security_posture)

### CODE REVIEW

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_10/data.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+metadata:
+  id: 43d5538c-17a3-5e04-9c06-ad4323bfd188
+  name: Ensure FTP deployments are Disabled
+  profile_applicability: '* Level 1'
+  description: |-
+    By default, Azure Functions, Web, and API Services 
+    can be deployed over FTP.
+    If FTP is required for an
+    essential deployment workflow, FTPS should be required
+    for FTP login for all App Service Apps and Functions.
```
Comment (oren-zohar) [inline security-policies/bundle/compliance/cis_azure/rules/cis_9_10/data.yaml:10]: Why are there randomly placed new lines? I wonder I it looks in Kibana

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_10/rule.rego [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	)
+}
+
+is_ftp_disabled {
+	is_ftp_enabled != true
+} else = false
+
+is_ftp_enabled {
+	data_adapter.site_config.ftpsState == "AllAllowed"
+} else = false
```
Comment (oren-zohar) [inline security-policies/bundle/compliance/cis_azure/rules/cis_9_10/rule.rego:23]: Why did you have to negate the result of `is_ftp_enabled`? And not implement the negated check in `is_ftp_disabled`?

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_10/rule.rego [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	data_adapter.is_website_asset
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(is_ftp_disabled),
+		data_adapter.resource,
+	)
+}
+
+is_ftp_disabled {
```
Comment (oren-zohar) [inline security-policies/bundle/compliance/cis_azure/rules/cis_9_10/rule.rego:17]:
```suggestion
is_ftps_disabled {
```

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+is_tls_version(version) {
+	data_adapter.site_config.minTlsVersion == version
+} else = false
+
+is_tls_configured(version) = r {
+	data_adapter.site_config.minTlsVersion
+	r = is_tls_version(version)
+}
```
Comment (oren-zohar) [inline security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego:24]: unused?

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(is_tls_version("1.2")),
+		data_adapter.resource,
+	)
+}
+
+is_tls_version(version) {
+	data_adapter.site_config.minTlsVersion == version
```
Comment (oren-zohar) [inline security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego:18]: what if the minTlsVersion is > 1.2? should we pass?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego (hunk 5 above)
Comment (jeniawhite) [reply security-policies/bundle/compliance/cis_azure/rules/cis_9_3/rule.rego:18]: Screenshot 2023-12-04 at 10 32 35

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_10/data.yaml (hunk 1 above)
Comment (jeniawhite) [reply security-policies/bundle/compliance/cis_azure/rules/cis_9_10/data.yaml:10]: This is auto-generated from the CIS provided benchmarks, let's see and it if it problematic we will need to change the behavior of parsing the CIS benchmark

File: security-policies/bundle/compliance/cis_azure/rules/cis_9_10/rule.rego (hunk 2 above)
Comment (jeniawhite) [reply security-policies/bundle/compliance/cis_azure/rules/cis_9_10/rule.rego:23]: Was easier to implement for use case when `data_adapter.site_config.ftpsState` property doesn't exist and we need to evaluate as pass

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**